src/modules/45-dashboards/pages/DashboardsCommonHeaderPage.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useState } from 'react'
+
+import { Color, Layout, Text, Icon } from '@wings-software/uicore'
+
+import { NavLink, useParams } from 'react-router-dom'
+import { Page } from '@common/exports'
+import { Breadcrumbs } from '@common/components/Breadcrumbs/Breadcrumbs'
+import routes from '@common/RouteDefinitions'
+import { useStrings } from 'framework/strings'
+import { GetStarted } from './home/GetStarted'
+import { useDashboardsContext } from './DashboardsContext'
+import css from './home/HomePage.module.scss'
+
+const DashboardsHeader: React.FC = () => {
+  const { getString } = useStrings()
+  const { breadcrumbs } = useDashboardsContext()
+  const { accountId, folderId } = useParams<{ accountId: string; folderId: string }>()
+  const [isOpen, setDrawerOpen] = useState(false)
+
+  const title = breadcrumbs ? breadcrumbs[breadcrumbs.length - 1].label : ''
```

Comment:
Looking at the interface of the context hook, breadcrumbs will always be an array. Do you mean to check if that array has items?
```suggestion
  const title = breadcrumbs.length ? breadcrumbs[breadcrumbs.length - 1].label : ''
```